BlueprintUICommonControls/Sources/AttributedLabel.swift

```diff
@@ -529,6 +553,17 @@ extension AttributedLabel {
             attributedText = mutableString
         }
 
+        override func accessibilityActivate() -> Bool {
```

Comment:
Previously, activating the element would send a touch event to the label's accessibility activation point which defaults to the center of the view. This would occasionally activate a link that happened to overlap that particular position.

We now have the provide behavior:
- No links: Not interactive, no effect.
- Exactly one link: Activate the link.
- More than one link: Ambiguous selection, no effect.
as always we additionally allow link selection and activation through the rotor.

Reply:
It would be good to capture this intent in a comment on the method, IMO